### ISSUE TITLE
Maintenance mode

### ISSUE THREAD

**Issue by Kenny-80** [nilsteampassnet/TeamPass#1962]
### Steps to reproduce
1. access http://server/teampass
2. type login password
3. bad password

### Expected behaviour
Maintenance mode should not be enabled
I have changed teampass_table maintenance_mode to 0

### Actual behaviour
I can't logon

### Server configuration
**Operating system**:
Centos 7.4.1708

**Web server:**
Apache 2.4.6

**Database:**
mysql ver 15.1 distrib 5.5.56-MariaDB

**PHP version:**
7.1.8

**Teampass version:** 
2.1.27

**Teampass configuration file:**

**Updated from an older Teampass or fresh install:**
Updated from 2.1.25

PLEASE attach to this message the file `/includes/config/tp.config.php`.
[tp.config.zip](https://github.com/nilsteampassnet/TeamPass/files/1374738/tp.config.zip)
### Client configuration

**Browser:**
Firefox 52.4 ESR

**Operating system:**
Windows 10

#### Web server error log
```
Insert your webserver log here
```

#### Log from the web-browser developer console (CTRL + SHIFT + i)

POST 
XHR 
http://10./teampass/sources/identify.php [HTTP/1.1 200 OK 630 ms]
GET 
http://10./teampass/includes/js/jquery-ui/images/ui-bg_glass_100_f8f8f8_1x400.png [HTTP/1.1 200 OK 0 ms]
GET 
http://10./teampass/includes/js/jquery-ui/images/ui-bg_inset-hard_75_999999_1x100.png [HTTP/1.1 200 OK 0 ms]
POST 
XHR 
http://10./teampass/sources/identify.php [HTTP/1.1 200 OK 28 ms]
```
Insert the log here and especially the answer of the query that failed.
```
2017-10-11T10:14:35.681563+02:00 localhost apache: [Wed Oct 11 10:14:35.680683 2017] [php7:warn] [pid 13908] [client 10.245.64.204:62472] PHP Warning:  session_start(): Failed to read session data: user (path: /var/lib/php/session) in /var/www/html/teampass/includes/libraries/csrfp/libs/csrf/csrfprotector.php on line 107
2017-10-11T10:14:35.682088+02:00 localhost apache: [Wed Oct 11 10:14:35.680814 2017] 

[php7:error] [pid 13908] [client 10.245.64.204:62472] PHP Fatal error:  Uncaught RuntimeException: Authentication failed in /var/www/html/teampass/sources/SecureHandler.php:125\nStack trace:\n#0 /var/www/html/teampass/sources/SecureHandler.php(63): PHPSecureSession\\SecureHandler->decrypt('phpCAS|a:1:{s:4...', '\\xFD\\x8A\\xB9\\x89L\\xA6\\x94G\\xE91\\xB2\\xB1\\x05c\\x93...')\n#1 [internal function]: PHPSecureSession\\SecureHandler->read('ST-742-lZhKyWmv...')\n#2 /var/www/html/teampass/includes/libraries/csrfp/libs/csrf/csrfprotector.php(107): session_start()\n#3 /var/www/html/teampass/index.php(45): csrfProtector::init()\n#4 {main}\n  thrown in /var/www/html/teampass/sources/SecureHandler.php on line 125

**Comment by nilsteampassnet** [comment]
I don't understand.

Maintenance mode is enabled after install. It can be changed by admin.

I don't see why to change it through DB.

**Comment by Kenny-80** [comment]
I can't logon with admin or administrator login, it always says bad password, so i can't take off the maintenance mode.
I'm pretty sure of the password, I don't understand why i can't connect

**Comment by nilsteampassnet** [comment]
I believe it is a duplicate of #1965

Please reinstall using new 2.1.27.9 package by over-writing existing files and performing new install process.

**Comment by Kenny-80** [comment]
It's not resolved, and i can't download 2.1.27.9, so the easier for me is to uninstall and never re-install.
Thanks.
